content/20.zksync-protocol/75.evm-emulator/04.technical-details.md

```diff
@@ -24,24 +25,29 @@ When an EVM contract is called, the following steps occur:
    - The system retrieves the contract’s **versioned hash** from `AccountCodeStorage`.
    - If the version is `0x02` (EVM contract), execution is routed through the **EVM Emulator**.
 
-2. **Bytecode Interpretation**
+1. **Bytecode Interpretation**
    - The emulator loads the contract’s EVM bytecode.
    - It initializes execution context (**EVM stack, memory, execution frames context**).
    - The emulator derives available amount of EVM gas from passed EraVM native gas (ergs).
    - The emulator interprets and executes opcodes in a loop. Every opcode consumes emulated EVM gas. Emulation itself consumes native EraVM gas (ergs).
    - Execution continues until completion, error, or out-of-ergs condition.
 
-3. **External Calls**
-   - If external call to another EVM contract is requested, EVM emulator pushes new frame in EvmGasManager and invokes new instance of the EVM emulator for that contract.
-   - If external call to EraVM contract is requested, EVM emulator calls that contract directly (EraVM gas is derived from passed EVM gas). When that call returns or fails, EVM emulator continues execution normally. Delegatecall to EraVM contracts is forbidden.
+1. **External Calls**
```

Comment:
Looks like numeration is broken

Reply:
actually no. We use `1.` for all but Nuxt takes care of showing the correct one 😉